### PR TITLE
Fix bug 

### PR DESCRIPTION
**Pull request by takara9**
Eliminate double quotation in the error message to prevent JSON broken

### CODE REVIEW

File: bmc-log-collector/log-collector.go [hunk 1]
```diff
@@ -80,10 +82,12 @@ func (c *selCollector) collectSystemEventLog(ctx context.Context, m Machine, log
 		}
 		lastPtr.LastHttpStatusCode = statusCode
 
-		if err != nil && lastPtr.LastError != err {
+		// Eliminate double quotation in the error message to prevent JSON broken.
+		errString := fmt.Sprintln(strings.Replace(fmt.Sprintf("%v", err), "\"", "", -1))
+		if err != nil && lastPtr.LastError != errString {
```
Comment (itchyny) [inline bmc-log-collector/log-collector.go:87]:
```suggestion
		if err != nil && lastPtr.LastError != err.Error() {
```

File: bmc-log-collector/log-collector.go [hunk 2]
```diff
@@ -70,21 +70,30 @@ func (c *selCollector) collectSystemEventLog(ctx context.Context, m Machine, log
...
-	if statusCode != 200 || err != nil {
+	if err != nil {
 		// Increment the failed counter
 		counterRequestFailed.WithLabelValues(m.Serial).Inc()
-
-		// Prevent log output by the same error code or httpStatus
-		if statusCode != 200 && statusCode != lastPtr.LastHttpStatusCode {
+		// Prevent log output by the same error code
+		if lastPtr.LastError != err.Error() {
+			slog.Error("failed to write a pointer file.", "err", err, "serial", m.Serial, "ptrDir", c.ptrDir)
```
Comment (masa213f) [inline bmc-log-collector/log-collector.go:78]: This error message is not appropriate.

**Review by masa213f — CHANGES_REQUESTED** [review]

File: bmc-log-collector/log-collector.go (hunk 2 above)
Comment (takara9) [reply bmc-log-collector/log-collector.go]: I am so sorry, change it.

**Review by masa213f — APPROVED** [review]
LGTM

**Pull request merged**